joomla/joomla-cms
# [com_templates] Support plugin layout overrides

Pull Request for Issue #22245 .

### Summary of Changes

This adds support for plugin layout overrides in Template Manager.

### Testing Instructions

Apply patch.
Go to template details. Switch to `Create Overrides` tab.
Check that plugins appear.
Check that overrides are created correctly.

### Documentation Changes Required
IDK.
